MRN-Code/coinstac
# Enhancement/issues 218 219

*  **Problems:**
    * Status-item button easily overlooked
    * Status-item button takes user to top of consortium page, rather than results section as expected
* **Solution:**
   * In addition to existing link, Bootstrap button added to bottom-right of status-item
   * Status-item link includes computation id to automatically expand the selected computation
   * ID added to results section to bring user directly to results section
* **Tests:**
   1. Login into application and note placement of new buttons in completed status items listed on home screen
   1. Click one of the newly added buttons and check that not only does the next page bring you to the **Results** section, but, once the data resolves, the selected computation is automatically expanded.
